Feature/docker odoo (#1)
focusate/docker-odoo: docs/addons-path.rst

Addons Path
###########

> Odoo includes paths for addons path from two directories by default: :code:`/opt/odoo/projects` and :code:`/mnt/extra-addons`. Paths are are included on runtime and recursively.

Reviewer: ```suggestion
Odoo includes paths for addons path from two directories by default: :code:`/opt/odoo/projects` and :code:`/mnt/extra-addons`. Paths are included on runtime and recursively.
```

> For example if addons directory is added in :code:`/opt/odoo/projects` and it also has sub directory that it also consists of Odoo addons, both of these paths will be included when loading Odoo.

Reviewer: ```suggestion
For example if addons directory is added in :code:`/opt/odoo/projects` and it has a sub directory, containing Odoo addons, both of these paths will be included when loading Odoo.
```
